src/com/tutorial/main/BasicEnemy.java

```diff
@@ -1,40 +0,0 @@
-package com.tutorial.main;
-
-import java.awt.Color;
-import java.awt.Graphics;
-import java.awt.Rectangle;
-
-public class BasicEnemy extends GameObject{
```

Comment:
One thing to consider is scope and access. Since this class isn't a superclass (doesn't have any classes that extend it), we can mark it as final with `public final class`. It's unnecessary in this project since everything is going to be contained within the project. However if you're creating some sort of an API/library/etc, You want to indicate to the implementers what they can and cannot do with the object.